### ISSUE TITLE
about RRACH sites 

### ISSUE THREAD

**Issue by xieyy46** [novoalab/EpiNano#117]
Hi developer！As your paper and github show Epinano trained the SVM model use 5-mer that contained only one A, should I discard the kmer in the results file with more than two As ？I saw in main page of Epinano github that you remind us to retain only RRACH sites in the result file, but some RRACH sites like GGACA contain more than one A. I am so confused. Thank you!
![image](https://user-images.githubusercontent.com/76148123/167096164-8e96a717-88b0-4b7a-9d73-019f0001f8a4.png)
![image](https://user-images.githubusercontent.com/76148123/167096235-632c81c5-e746-4006-8852-e5b2136e6102.png)
![image](https://user-images.githubusercontent.com/76148123/167097073-c7943329-527c-428e-976c-0a6851329649.png)

**Comment by enovoa** [comment]
Hi @xieyy46 , the model is trained with sites that do not contain two As in the same k-mer, because otherwise both would be modified (which is not the case in biological situations, as per what we know). For this reason, we recommend to exclude k-mers with 2As, as those have not been explicitely trained or tested in the SVM model.
However that being said, we did test the performance of the model in all RRACH sites, which contain, as you mention kmers such as GGACA. Since at the end the final model only considers information from the middle position of the kmer (q3.mis3.del3), in our paper we included GGACA k-mers in our analysis of the performance of the algorithm. 

Hope that clarified your doubts.

Finally I'd like to note that the SVM on single sample is not recommended, and that EpiNano-Error overperforms EpiNano-SVM in our hands.  

Thanks,
Eva
